src/components/User/Operations/OperationDetails.vue

```diff
@@ -1,9 +1,11 @@
 <template>
-  <div class="operation-details">
-    <div class="app__block">
-      <h2>Operation details</h2>
-      <div class="operation-details__op-details-wrp">
-        <details-reader :details="operation" />
+  <div>
+    <div class="operation-details">
+      <div class="app__block">
+        <h2>Operation details</h2>
+        <div class="operation-details__op-details-wrp">
+          <details-reader :details="operation" />
```

Comment:
Check that it will work fine until data loaded. Use data loading throttle in the Chrome Network panel